cmake/onnxruntime_hosting.cmake

```diff
@@ -27,13 +27,43 @@ find_package(Boost 1.69 COMPONENTS system context thread program_options REQUIRE
 set(re2_src ${REPO_ROOT}/cmake/external/re2)
 
 # Setup source code
+file(GLOB_RECURSE onnxruntime_hosting_lib_srcs
+        "${ONNXRUNTIME_ROOT}/hosting/util.cc"
+        )
 file(GLOB_RECURSE onnxruntime_hosting_srcs
-    "${ONNXRUNTIME_ROOT}/hosting/*.h"
-    "${ONNXRUNTIME_ROOT}/hosting/*.cc"
+    "${ONNXRUNTIME_ROOT}/hosting/main.cc"
```

Comment:
Could you format this whole file consistently?

Reply:
Done